proof/crefine/lib/Boolean_C.thy

```diff
@@ -0,0 +1,140 @@
+(*
+ * Copyright 2023, Proofcraft Pty Ltd
+ * Copyright 2014, General Dynamics C4 Systems
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ *)
+
+(* Definitions and rules for C boolean constants *)
```

Comment:
not sure if it's useful saying "parsed from the kernel" or something, as they're boolean constants defined in the seL4 C code itself:
```
enum _bool {
    false = 0,
    true = 1
};
```

Reply:
Done